web/admin/src/Admin.js

```diff
@@ -110,17 +115,22 @@ export default class Admin extends PureComponent {
   onSessionNameChange = e => this.sessionsRef().child(this.state.sessionId).update({name: e.target.value})
   onBackgroundUrlChange = e => this.backgroundUrlRef().set(e.target.value)
   onSecondsChange = e => this.sessionsRef().child(this.state.sessionId).update({secondsPerQuestion: +e.target.value})
-  createSession = () => this.sessionsRef().push({name: 'New Session', secondsPerQuestion: 30}).then(ref => this.setState({sessionId: ref.key}))
+  createSession = () => this.sessionsRef().push({name: 'New Session', secondsPerQuestion: 30}).then(ref => {
+    this.setState({sessionId: ref.key})
+    this.addQuestion(ref.key)
+  })
   deleteSession = () => {
     const {sessionId, sessions} = this.state
+    const publicSessions = Object.values(this.state.publicSessions)
```

Comment:
Is `publicSession.id` the same as `sessionId`?  If so, we don't need to look up the id of the publicSession.

Reply:
Unfortunately the keys are not the same only the session name. Should we store the key with the public session?